perfkitbenchmarker/linux_benchmarks/cloudharmony_network_benchmark.py

```diff
@@ -110,6 +119,12 @@ def CheckPrerequisites(_):
   # TODO(user): add AWS & Azure support for object storage
   if FLAGS.ch_network_test_service_type == STORAGE and FLAGS.cloud != 'GCP':
     raise NotImplementedError('Benchmark only supports GCS object storage.')
+  elif FLAGS.ch_network_test_service_type == DNS and FLAGS.ch_network_test != 'dns':
```

Comment:
You can combine these two checks. 
elif FLAGS.ch_network_test_service_type != FLAGS.ch_network_test

Reply:
Combined the two checks. I did it slightly differently because this case is specific to the DNS test